changelog/fix_gem_version_not_respecting_tags.md

```diff
@@ -0,0 +1 @@
+* [#9803](https://github.com/rubocop/rubocop/pull/9803): Fix `Bundler/GemVersion` cop not respecting git tags. ([@tejasbubane,@timlkelly][])
```

Comment:
I am not sure if the syntax is correct here for multiple contributors but I wanted to make sure to include both of our work here. Should this be different?

Reply:
Each contributor would have a separate link. This should work:

```suggestion
* [#9803](https://github.com/rubocop/rubocop/pull/9803): Fix `Bundler/GemVersion` cop not respecting git tags. ([@tejasbubane][], [@timlkelly][])
```

Reply:
Ah, got it. Makes sense, thank you!